src/python/tests/test_configuration_manager.py

```diff
@@ -0,0 +1,560 @@
+"""
+The following tests the configuration manager component For the most part,
+the code uses the boolean type for testing due to simplicity; but contains
+tests to verify which data_types are supported and which are not.
+"""
+
+import asyncio
+import os
+import pytest
+import sqlalchemy as sa
+import aiopg.sa
+from foglamp.configuration_manager import (create_category, set_category_item_value_entry,
+                                           register_interest, get_all_category_names,
+                                           get_category_all_items, get_category_item,
+                                           get_category_item_value_entry, _registered_interests,
+                                           _configuration_tbl)
+__author__ = "Ori Shadmon"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+_CONNECTION_STRING = "dbname='foglamp'"
+
+pytestmark = pytest.mark.asyncio
+
+async def delete_from_configuration():
+    """Remove initial data from configuration table"""
+    async with aiopg.sa.create_engine(_CONNECTION_STRING) as engine:
+         async with engine.acquire() as conn:
+             await conn.execute(_configuration_tbl.delete())
```

Comment:
@oshadmon  I think this PR can be merged; **if you delete the stuff which you created only.** instead of DELETE all; as we have no provision yet to reset db and actually use test_foglamp db for test. I think for you can have a list of key in data (make it module var) and delete them all.

Reply:
also please avoid db execute in loop; seems may of them got fixed. But I still see few references